packages/client/wallets/smart-wallet/src/error/index.ts

```diff
@@ -119,6 +120,18 @@ export class UserWalletAlreadyCreatedError extends SmartWalletSDKError {
     }
 }
 
+export class PasskeyPromptError extends SmartWalletSDKError {
+    public passkeyName: string;
+
+    constructor(passkeyName: string) {
+        super(
+            `Prompt was either cancelled or timed out for passkey ${passkeyName}`,
+            SmartWalletErrors.ERROR_PASSKEY_PROMPT
```

Comment:
There should be an `undefined` here that's inbetween.

I fixed in this PR: https://github.com/Crossmint/crossmint-sdk/pull/633